lib/plugins/aws/package/compile/events/stream/index.js

```diff
@@ -44,6 +44,7 @@ class AwsCompileStreamEvents {
             let BatchSize = 10;
             let StartingPosition = 'TRIM_HORIZON';
             let Enabled = 'True';
+            let Timestamp = null;
```

Comment:
I would vote to rename this to `StartingPositionTimestamp` (even if the name might be verbose).

The reason for that is that all other config parameters match the AWS config as well (http://docs.aws.amazon.com/lambda/latest/dg/API_CreateEventSourceMapping.html#API_CreateEventSourceMapping_RequestSyntax).